### PR TITLE
minor scrub/load fixes

### PR DESCRIPTION
**Pull request by mwarin**
Addressing:
* bin/process_new_holdings.sh did not recognize .gz files in data/new/
* issues with nil values in records that had fewer columns than the header
* lib/autoscrub.rb calculating the wrong batch size for .gz files

### CODE REVIEW

**Review by aelkiss — APPROVED** [review]
:+1:

File: lib/member_holding.rb [hunk 1]
```diff
@@ -52,7 +52,10 @@ def parse(str)
   # rubocop:disable Metrics/MethodLength
   def set(col_type, col_val)
     if col_val.nil?
-      raise ColValError, "col_val for col_type #{col_type} is nil"
+      Services.scrub_logger.warn(
```
Comment (aelkiss) [inline lib/member_holding.rb:55]: Maybe worth a test that it warns but doesn't raise?

Comment (mwarin) [reply lib/member_holding.rb:55]: Sure.

Comment (mwarin) [reply lib/member_holding.rb:55]: Added https://github.com/hathitrust/holdings-backend/pull/92/commits/579c3cf38d7de7dd0518ffbf97e9e140f897ce23

**Pull request merged**